fix(core): avoid logging which bootstrap node is being connected to

anthonybilinski opened the pull request:
Errors are parsed and printed, but which specific node is being connected to
isn't very relevant to any errors that would occur, and the nodes list is
already updated to prune offline nodes based on nodes.tox.chat periodically.
This provides some extra privacy about which connections are being made,
even though the bootstrap nodes are already public.

- [x] Commits follow our [git commit guidelines](https://github.com/qTox/qTox/blob/master/CONTRIBUTING.md#git-commit-guidelines)

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/qtox/qtox/5865)
<!-- Reviewable:end -->

anthonybilinski committed:
chore(tools): allow merging PR to whatever branch is checked out

Useful for either dev testing or merging PRs to dev branches. Also stop
updating local repo state beyond merging the PR.

anthonybilinski committed:
chore(travis): make travis happy by having some output - but not too much

Fix #5515

brew install might take a long time to build Qt or ffmpeg. Travis kills us if
we don't output for 10 minutes. Travis also kills us if we output too much, so
verbose isn't an option. So just output some dots...

anthonybilinski committed:
fix(core): avoid logging which bootstrap node is being connected to

Errors are parsed and printed, but which specific node is being connected to
isn't very relevant to any errors that would occur, and the nodes list is
already updated to prune offline nodes based on nodes.tox.chat periodically.
This provides some extra privacy about which connections are being made,
even though the bootstrap nodes are already public.

Kribylet reviewed (commented):
<img class="emoji" title=":lgtm_strong:" alt=":lgtm_strong:" align="absmiddle" src="https://reviewable.io/lgtm_strong.png" height="20" width="61"/>

> Reviewed 1 of 1 files at r1.
> *[Reviewable](https://reviewable.io/reviews/qtox/qtox/5865#-:-LqeuUzdBWJZU5IeQUHE:b-jbsf8z)* status: :shipit: complete! 1 of 1 LGTMs obtained



<!-- Sent from Reviewable.io -->

sphaerophoria reviewed (commented):
<img class="emoji" title=":lgtm_strong:" alt=":lgtm_strong:" align="absmiddle" src="https://reviewable.io/lgtm_strong.png" height="20" width="61"/>

> Reviewed 1 of 1 files at r1.
> *[Reviewable](https://reviewable.io/reviews/qtox/qtox/5865#-:-Lqillhu3ON5mQrAGNCe:b-jbsf8z)* status: :shipit: complete! 2 of 1 LGTMs obtained



<!-- Sent from Reviewable.io -->

anthonybilinski committed:
fix(ci): upgrade travis to xcode9.3 to get macos 10.13 with brew support

Brew now only officially supports Mac OS 10.13-10.15, which means our
previously used 10.12 had fewer binary packages available, meaning we had to
compile more ourselves, causing CI to timeout.
https://docs.travis-ci.com/user/reference/osx#macos-version
https://github.com/Homebrew/brew/pull/6500

Fix #5515

anthonybilinski committed:
fix(osx): create /usr/local/sbin directory for brew package installation

ffmpeg dependency unbound attempts to symlink to /usr/local/sbin, failing if
the directory does not exist. /usr/local/sbin does not exist by default on
10.13 High Sierra, so prompt to create it if it doesn't exit, and create it for
travis CI automatically.
